griptape/utils/chat.py

```diff
@@ -4,9 +4,10 @@
 
 if TYPE_CHECKING:
     from griptape.structures import Structure
+from griptape.utils import Stream
 
 
-@define(frozen=True)
+@define
```

Comment:
What is the reason for removing frozen?

Reply:
In order to be able to wrap structure in Stream once rather than during each conversation turn. I saw your comments below regarding not reassigning so I'll revert this.